PicPi/AddPhotoViewController.swift

```diff
@@ -0,0 +1,196 @@
+//
+//  AddPhotoViewController.swift
+//  PicPi
+//
+//  Created by Mustafa T. Mohammed on 12/15/20.
+//
+import Photos
+import Moya
+import UIKit
+import BSImagePicker
+
+class AddPhotoViewController: UIViewController {
+  
+  let sendButton = customBtnRoundCornerBlueWithShadow(type: .custom)
+  let photoUploadManager = PhotoUploadManager()
+  let doneButton = customBtnRoundCornerBlueWithShadow(type: .custom)
+  
+  private let tableView = UITableView()
+  private var margins: UILayoutGuide!
+  
+  override func loadView() {
+    super.loadView()
+    view.backgroundColor = .white
+    margins = view.layoutMarginsGuide
+    setupSendButton()
+    setupDoneBtn()
+    setupTableView()
+    photoUploadManager.delegate = self
+    
+  }
+  
+  // MARK: - UI Setup
+  private func setupSendButton() {
+    
+    view.addSubview(sendButton)
+    sendButton.addTarget(self, action: #selector(sendButtonPressed), for: .touchUpInside)
+    sendButton.setTitle("Send", for: .normal)
+    sendButton.setTitleColor(.darkGray, for: .disabled)
+    sendButton.isEnabled = false
+    sendButton.sizeToFit()
```

Comment:
i don't believe this is needed when working with constraints

Reply:
if I don't use sizeToFit I need to specify the button height and width like     sendButton.frame.size.height = 34
what do you suggest?

Reply:
Ooh, strange, it should resize on its own when you see the title

We can leave it as is